checker/src/org/checkerframework/checker/interning/qual/Interned.java

```diff
@@ -26,16 +28,18 @@
  * annotated.
  * <p>
  *
- * This annotation is associated with the {@link InterningChecker}.
+ * This annotation is associated with the {@link org.checkerframework.checker.interning.InterningChecker}.
```

Comment:
OK.  If your IDE cannot handle this, it's OK to make the change, but I would do so in a different commit with a clear commit message.  Thanks!